src/runtime/virtcontainers/virtiofsd.go

```diff
@@ -123,6 +133,8 @@ func (v *virtiofsd) Start(ctx context.Context, onQuit onQuitFunc) (int, error) {
 		return pid, err
 	}
 
+	go io.Copy(logWriter, stderr)
```

Comment:
Since the following go routine also read the virtiofsd's stderr, I'm afraid we can merge them into one.

Reply:
```
diff --git a/src/runtime/virtcontainers/virtiofsd.go b/src/runtime/virtcontainers/virtiofsd.go
index bb2035e3..7cc075e2 100644
--- a/src/runtime/virtcontainers/virtiofsd.go
+++ b/src/runtime/virtcontainers/virtiofsd.go
@@ -133,10 +133,11 @@ func (v *virtiofsd) Start(ctx context.Context, onQuit onQuitFunc) (int, error) {
                return pid, err
        }
 
-       go io.Copy(logWriter, stderr)
-
-       // Monitor virtiofsd's stderr and stop sandbox if virtiofsd quits
+       // Copy virtiofsd's stderr to the log file, while also monitoring
+       // it to stop the sandbox if virtiofsd quits for some reason
        go func() {
+               io.Copy(logWriter, stderr)
+
                scanner := bufio.NewScanner(stderr)
                for scanner.Scan() {
                        v.Logger().WithField("source", "virtiofsd").Info(scanner.Text())

```

Like this?  If you agree with that, I'll update the PR.

Reply:
Hi @fidencio 

I think we can delete the following output to logger codes:
- scanner := bufio.NewScanner(stderr)
-                for scanner.Scan() {
-                        v.Logger().WithField("source", "virtiofsd").Info(scanner.Text())

Reply:
Thanks, @lifupan, I've updated the PR accordingly.